### ISSUE TITLE
installing python dependancies not working for PAT v0.3.0

### ISSUE THREAD

**Issue by cratcliff** [CSIRO-Precision-Agriculture/PAT_QGIS_Plugin#32]
I have been attempting to install the PAT plugin in QGIS version 2.18.26 by closely following the user manual. I'm having issues with the python package installer, Install_PAT_Extras, and I'm unsure what to try next. When reopening QGIS, a message appears saying "Installation or updates are required for fiona, pyprecag, rasterio. Please quit QGIS and run Install_PAT_Extras located on your desktop". I have tried this multiple times with no success. Any directions on how to fix this would be appreciated.

**Comment by cratcliff** [comment]
There are two reasons this could occur. 
1. The user name has spaces in it. 
2. The pat folder no longer exists in the qgis plugin folder.

**Comment by cratcliff** [comment]
Install failing if desktop installation link doesn't exist.
```
Couldn't load plugin pat due to an error when calling its classFactory() method 
WindowsError: [Error 2] The system cannot find the file specified: 'C:\\Users\\XXX\\Desktop\\Install PAT Extras.lnk' 

Traceback (most recent call last):
  File "C:/PROGRA~1/QGIS2~1.18/apps/qgis-ltr/./python\qgis\utils.py", line 333, in startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "C:/Users/atareque/.qgis2/python/plugins\pat\__init__.py", line 117, in classFactory
    check_python_dependencies(PLUGIN_DIR, iface)
  File "C:/Users/XXX/.qgis2/python/plugins\pat\util\check_dependencies.py", line 320, in check_python_dependencies
    os.remove(os.path.join(shortcutPath))
WindowsError: [Error 2] The system cannot find the file specified: 'C:\\Users\\XXX\\Desktop\\Install PAT Extras.lnk'
```

**Comment by cratcliff** [comment]
It's still an issue with 0.3.1. 
`Install PAT Extras.lnk` still not on desktop.
Sometimes getting a `custwidg` error

**Comment by cratcliff** [comment]
Can't create logs if  `<user>/.qgis/python/plugins/pat` doesn't exist.

**Comment by cratcliff** [comment]
Fixed in PAT v0.3.2

**Comment by ArgaHita** [comment]
![Screenshot (40)](https://user-images.githubusercontent.com/115727156/195643745-928d682b-954d-4608-b571-652a267c7505.png)

How to solve it? need hurry up help. thx all
